### PR TITLE
feat(24.04): add httping slices

### PR DESCRIPTION
**Pull request by hd9**
# Proposed changes
This PR adds the `httping` slice to Ubuntu 24.04, a ping-like program for HTTP requests.

## Related issues/PRs
#277
#278 

### Forward porting
N/A

## Testing
```bash

fs=testfs
rm -rf ./$fs; mkdir ./$fs
chisel cut --release ./chisel-releases/ --root $fs httping_bins
sudo chroot $fs /usr/bin/httping --version


2022024/07/05 18:39:47 Processing ./chisel-releases/ release...
2024/07/05 18:39:48 Selecting slices...
2024/07/05 18:39:48 Fetching ubuntu 24.04 noble suite details...
2024/07/05 18:39:50 Release date: Thu, 25 Apr 2024 15:10:33 UTC
2024/07/05 18:39:50 Fetching index for ubuntu 24.04 noble main component...
2024/07/05 18:39:50 Fetching index for ubuntu 24.04 noble universe component...
2024/07/05 18:39:50 Fetching ubuntu 24.04 noble-security suite details...
2024/07/05 18:39:51 Release date: Fri, 05 Jul 2024 12:42:08 UTC
2024/07/05 18:39:51 Fetching index for ubuntu 24.04 noble-security main component...
2024/07/05 18:39:51 Fetching index for ubuntu 24.04 noble-security universe component...
2024/07/05 18:39:51 Fetching ubuntu 24.04 noble-updates suite details...
2024/07/05 18:39:51 Release date: Sat, 06 Jul 2024  0:10:59 UTC
2024/07/05 18:39:51 Fetching index for ubuntu 24.04 noble-updates main component...
2024/07/05 18:39:52 Fetching index for ubuntu 24.04 noble-updates universe component...
2024/07/05 18:39:52 Fetching pool/main/b/base-files/base-files_13ubuntu10_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/g/gcc-14/gcc-14-base_14-20240412-0ubuntu1_amd64.deb...
2024/07/05 18:39:52 Fetching pool/universe/h/httping/httping_2.5-5.2build2_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/g/glibc/libc6_2.39-0ubuntu8.2_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/f/fftw3/libfftw3-double3_3.3.10-1ubuntu3_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/g/gcc-14/libgomp1_14-20240412-0ubuntu1_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/n/ncurses/libtinfo6_6.4+20240113-1ubuntu2_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/n/ncurses/libncursesw6_6.4+20240113-1ubuntu2_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/o/openssl/libssl3t64_3.0.13-0ubuntu3.1_amd64.deb...
2024/07/05 18:39:52 Fetching pool/main/o/openssl/openssl_3.0.13-0ubuntu3.1_amd64.deb...
2024/07/05 18:39:52 Extracting files from package "base-files"...
2024/07/05 18:39:52 Extracting files from package "gcc-14-base"...
2024/07/05 18:39:52 Extracting files from package "httping"...
2024/07/05 18:39:52 Extracting files from package "libc6"...
2024/07/05 18:39:52 Extracting files from package "libfftw3-double3"...
2024/07/05 18:39:52 Extracting files from package "libgomp1"...
2024/07/05 18:39:52 Extracting files from package "libtinfo6"...
2024/07/05 18:39:52 Extracting files from package "libncursesw6"...
2024/07/05 18:39:52 Extracting files from package "libssl3t64"...
2024/07/05 18:39:52 Extracting files from package "openssl"...
HTTPing v2.5, (C) 2003-2016 folkert@vanheusden.com
 * SSL support included (-l)
 * ncurses interface with FFT included (-K)
 * TFO (TCP fast open) support included (-F)

```

## Checklist
* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

**Comment by cjdcordeiro** [comment]
for the CLA check pls try re-committing via the CLI

File: slices/libfftw3-double3.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+essential:
+  - libfftw3-double3_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgomp1_libs
+    contents:
+      /usr/lib/*-linux-gnu*/libfftw3*:
```
Comment (cjdcordeiro) [inline slices/libfftw3-double3.yaml:12]: Better be a bit more specific about this pattern

Comment (hd9) [reply slices/libfftw3-double3.yaml]: Pushed changes as requested

**Review by cjdcordeiro — APPROVED** [review]
lgtm thanks

File: slices/httping.yaml
Comment (cjdcordeiro) [inline slices/httping.yaml:1]: btw, I know we talked about this, but it's ok to keep it. it's redundant cause it's being delivered by the other packages, but I'd be fine with that fyi

**Review by zhijie-yang — APPROVED** [review]
LGTM.

**Pull request merged**